packages/ceramic-doctype-tile/src/tile-doctype.ts

```diff
@@ -74,7 +74,7 @@ export class TileDoctype extends Doctype {
         }
 
         let unique: string
-        if (metadata.isUnique) {
+        if (metadata.isUnique || !('isUnique' in metadata)) { // default isUnique to true
```

Comment:
Making the change at the low level like I did in this PR means that schema document creation behavior changes as well, is that what we want?  It kind of makes sense to me that schema documents would default to being global even if other types of tile documents default to being unique, what do you think @oed @simonovic86?

Reply:
I think it would be weird ro have different behavior for different document content. Would prefer consistent behavior. 
Also don't see an inherent reason for a different behavior for schemas.